scripts/pyperf_build.sh

```diff
@@ -5,7 +5,7 @@
 #
 set -e
 
-git clone --depth 1 -b v1.0.5 https://github.com/Granulate/bcc.git && cd bcc && git reset --hard a2d0ae75ce59813eb571e0019f6640d2ee539bce
+git clone --depth 1 -b v1.0.7 https://github.com/Granulate/bcc.git && cd bcc && git reset --hard 3ffa8982fdbfce183da81af634dc39f743e65d59
```

Comment:
We shouldn't deploy new PyPerf yet because we haven't fixed yet the issue with the bpf_probe_read_user_str return codes.

Reply:
But it's not new? Which issue are you referring to, exactly.

Reply:
It is new. Since https://github.com/Granulate/bcc/commit/af26c253f6421720efdb32a7230eae4032b14cf4 which is not in v1.0.5 - the currently deployed version.

Reply:
Given an error post https://github.com/Granulate/bcc/commit/af26c253f6421720efdb32a7230eae4032b14cf4, how would it appear before that patch?

Reply:
A valid frame might appear as an error.

Reply:
Woops, I meant the issue with module-level no-args functions (`get_classname`).

Reply:
Actually it seems like it would be ok. A bit fickle however.

Reply:
Besides of all issues that we *don't* have a solution for, all issues are solved

Reply:
#ציטוטים